scripts/tests/test_zkai_attention_kv_d16_native_width_gate.py

```
@@ -0,0 +1,168 @@
+            ("proof_version", gate.D8_PROOF_VERSION, "proof version drift"),
+            ("required_backend_version", gate.D8_REQUIRED_BACKEND_VERSION, "backend version drift"),
+            ("selected_positions", list(gate.D8_SELECTED_POSITIONS), "selected positions drift"),
+            ("timing_policy", "benchmark_median_of_5", "timing policy drift"),
+            ("envelope_size_bytes", gate.D16_PROOF_SIZE_BYTES + 1, "envelope-size scale drift"),
+        ):
+            mutated = copy.deepcopy(payload)
+            for field in ("mutation_cases", "mutations_checked", "mutations_rejected", "all_mutations_rejected"):
+                mutated.pop(field)
+            mutated["scaled_receipt"][key] = value
+            mutated["scale_gate_commitment"] = gate._expected_commitment(mutated)
+            with self.assertRaisesRegex(gate.AttentionKvD16NativeWidthGateError, message):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)